client/src/state/DeckBuilderModule.ts

```diff
@@ -113,10 +113,18 @@ export const deckBuilderStore = vuexModule(rootStore, MODULE_NAME, {
                 (<DraftCard[][]>Array(numColumns)).fill([]).map(() => []);
             for (let i = 0; i < numColumns; i++) {
               newSection[i] = cards.filter(card => {
-                if (i === numColumns - 1) {
-                  return card.definition.cmc >= i;
+                const isDfc = (card.definition.card_faces?.length || 0) > 0;
```

Comment:
So I've realized that card_faces is never actually null. I just merged a change to make its type not nullable, so you can lose the ? stuff if you want to